packages/upgrade/src/cli.js

```diff
@@ -5,96 +5,112 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-'use strict';
```

Comment:
Can the `use strict` be removed because this file is going through the `esbuild` process?

Reply:
Yeah! Definitely